src/index.html

```diff
@@ -9,6 +9,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div>
+        <a href="#home">
+          <img src="./images/logo.png" alt="moyo">
+        </a>
+      </div>
+      <nav>
+        <ul class="nav_list">
+          <li class="nav_item first_item">
+            <a href="#apple" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a href="#samsung" class="nav_link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="#smartphones" class="nav_link">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a
+            href="#laptops&computers"
+            class="nav_link"
+            data-qa="hover"
+            >
```

Comment:
will reject next time I see this
```suggestion
            <a
              href="#laptops&computers"
              class="nav_link"
              data-qa="hover"
            >
```

Reply:
Okay , got it!